src/main/java/wooteco/subway/domain/PathFinder.java

```diff
@@ -0,0 +1,51 @@
+package wooteco.subway.domain;
+
+import java.util.List;
+import org.jgrapht.GraphPath;
+import org.jgrapht.alg.shortestpath.DijkstraShortestPath;
+import org.jgrapht.graph.DefaultWeightedEdge;
+import org.jgrapht.graph.WeightedMultigraph;
+
+public class PathFinder {
+
+    final private DijkstraShortestPath<Station, DefaultWeightedEdge> dijkstraShortestPath;
+
+    public PathFinder(final List<Station> stations, final List<Section> sections) {
+        final WeightedMultigraph<Station, DefaultWeightedEdge> graph = new WeightedMultigraph(DefaultWeightedEdge.class);
+        addVertex(stations, graph);
+        addEdge(sections, graph);
+        this.dijkstraShortestPath = new DijkstraShortestPath(graph);
+    }
+
+    private void addEdge(final List<Section> sections, final WeightedMultigraph<Station, DefaultWeightedEdge> graph) {
+        for (final Section section : sections) {
+            graph.setEdgeWeight(graph.addEdge(section.getUpStation(), section.getDownStation()), section.getDistance());
+            graph.setEdgeWeight(graph.addEdge(section.getDownStation(), section.getUpStation()), section.getDistance());
```

Comment:
```suggestion
            DefaultWeightedEdge downEdge = graph.addEdge(section.getUpStation(), section.getDownStation());
            DefaultWeightedEdge upEdge = graph.addEdge(section.getDownStation(), section.getUpStation());
            graph.setEdgeWeight(downEdge, section.getDistance());
            graph.setEdgeWeight(upEdge, section.getDistance());
```
동작은 같은 코드인데요, 읽을 때 어떤 차이가 있을까요?

Reply:
해당 코드가 양 방향의 엣지를 모두 생성해준다는 것을 이해하기 더 쉬워질 것 같아요!!

더 가독성을 높일 수 있는 방법을 고려해봤어야 했는데, 예시 코드의 형태만을 가져다 써버렸네요..ㅎㅎ 반성하겠습니다!!